ios/ExpensifyCash/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0.1</string>
+	<string>1.0.7</string>
```

Comment:
We absolutely _do not_ want to commit any manual changes to any `Info.plist` file or `build.gradle` file.